iceoryx_dust/test/moduletests/test_container_fixed_position_container.cpp

```diff
@@ -92,6 +94,326 @@ TEST_F(FixedPositionContainer_test, Capacity)
     EXPECT_THAT(sut.capacity(), Eq(CAPACITY));
 }
 
+// BEGIN test copy constructor
+
+TEST_F(FixedPositionContainer_test, UsingCopyCtorEmptyContainerResultsInEmptyContainer)
+{
+    ::testing::Test::RecordProperty("TEST_ID", "6c528ef3-9c2d-4eb2-93a9-2d998d0db380");
+
+    SutComplex copy_sut_complex{sut_complex};
+
+    EXPECT_THAT(copy_sut_complex.empty(), Eq(true));
```

Comment:
Please also check `full` and `size` here. No need to create another test for this.